x-pack/plugin/core/src/test/java/org/elasticsearch/license/LicenseServiceTests.java

```diff
@@ -17,23 +17,11 @@
 /**
  * Due to changes in JDK9 where locale data is used from CLDR, the licence message will differ in jdk 8 and jdk9+
  * https://openjdk.java.net/jeps/252
+ * We run ES with -Djava.locale.providers=SPI,COMPAT and same option has to be applied when running this test from IDE
  */
 public class LicenseServiceTests extends ESTestCase {
 
-    public void testLogExpirationWarningOnJdk9AndNewer() {
-        assumeTrue("this is for JDK9+", JavaVersion.current().compareTo(JavaVersion.parse("9")) >= 0);
-
-        long time = LocalDate.of(2018, 11, 15).atStartOfDay(ZoneOffset.UTC).toInstant().toEpochMilli();
-        final boolean expired = randomBoolean();
-        final String message = LicenseService.buildExpirationMessage(time, expired).toString();
-        if (expired) {
-            assertThat(message, startsWith("LICENSE [EXPIRED] ON [THU, NOV 15, 2018].\n"));
-        } else {
-            assertThat(message, startsWith("License [will expire] on [Thu, Nov 15, 2018].\n"));
-        }
-    }
-
-    public void testLogExpirationWarningOnJdk8() {
+    public void testLogExpirationWarning() {
         assumeTrue("this is for JDK8 only", JavaVersion.current().equals(JavaVersion.parse("8")));
```

Comment:
@pgomulka I think this part of the change might have been incorrect...
We collapsed the 2 tests into 1, but that one test is set to only run on JDK8. I think this `assumeTrue` should be removed?

Reply:
@tvernum you are right, the same applies to 7.x branch as the output should be the same.
thanks for taking a look!